portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1092,6 +1092,55 @@ definition {
 		PortletEntry.publish();
 	}
 
+	@description = "This is a use case for LPS-110450."
+	@priority = "5"
+	test VerifyErrorMessageInSaas {
+		property osgi.app.includes = "saml-saas";
+		property test.name.skip.portal.instance = "SAML#ErrorMessageValidation";
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "SSO",
+			configurationName = "SAML Configuration",
+			configurationScope = "System Scope");
+
+		FormFields.disableCheckbox(fieldName = "Enable Identity Provider Role To Be Configured");
+
+		SystemSettings.saveConfiguration();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "Virtual Instances");
+
+		PortalInstances.addCP(
+			mailDomain = "www.able.com",
+			virtualHost = "www.able.com",
+			webId = "www.able.com");
```

Comment:
Is it required to create a new portal instance? Most system settings affect the default portal instance.

Reply:
In this setup yes, we would like to mimic a real scenario whit a virtual instance, but I test how it works without it as well.